src/Microsoft.DocAsCode.Common/Git/GitUtility.cs

```diff
@@ -13,6 +13,7 @@ namespace Microsoft.DocAsCode.Common.Git;
 
 public static class GitUtility
 {
+    private const string GitTimeoutEnvVarName = "DOVFX_GIT_TIMEOUT";
     private static readonly string CommandName = "git";
     private static readonly int GitTimeOut = 1000;
```

Comment:
It is a bit unintuitive to know if the static constructor runs first or this static assignment runs first, consider use a static method.

```suggestion
    private static readonly int GitTimeout = GetGitTimeout();
```